wire-runtime/src/test/java/com/squareup/services/RxJavaService2SomeEndpoints.java

```diff
@@ -59,4 +42,18 @@ public RxJavaService2SomeEndpoints(Endpoint endpoint) {
   public Func1<LetsDataRequest, LetsDataResponse> getLetsData() {
     return letsData;
   }
+
+  public interface Endpoint {
+    /**
+     * Sends some more data.
+     */
+    @POST("/com.squareup.services.RxJavaService2/SendSomeMoreData")
```

Comment:
Seems to be a bugfix since the remote service would be based on the proto name not the generated class name.